.circleci/config.yml

```diff
@@ -11,39 +11,32 @@ workflows:
           write_sbt_version: true
       - codacy/sbt:
           name: populate_cache
-          install_sbt_version: 1.3.3
           persist_to_workspace: true
-          cmd: sbt ";set scalafmtUseIvy in ThisBuild := false;+update"
+          cmd: sbt "set scalafmtUseIvy in ThisBuild := false;+update"
           requires:
             - codacy/checkout_and_version
       - codacy/sbt:
           name: check_format
-          install_sbt_version: 1.3.3
-          cmd: sbt ";scalafmt::test;test:scalafmt::test;sbt:scalafmt::test"
+          cmd: sbt "scalafmt::test;test:scalafmt::test;sbt:scalafmt::test"
           requires:
             - populate_cache
       - codacy/sbt:
           name: compile
-          install_sbt_version: 1.3.3
           persist_to_workspace: true
-          cmd: sbt ";+compile;+test:compile"
+          cmd: sbt "+compile;+test:compile"
           requires:
             - populate_cache
       - codacy/sbt:
           name: test
-          install_sbt_version: 1.3.3
           cmd: |
-            sbt ";coverage;test"
-            sbt ";coverageReport"
-            sbt ";coverageAggregate"
-            sbt ";codacyCoverage"
+            sbt "coverage;test;coverageReport;coverageAggregate"
```

Comment:
Coverage aggregate needs to be separated as stated here https://github.com/scoverage/sbt-scoverage#multi-project-reports

Reply:
I know we want to shorted stuff but we need to make sure we dont break these things.
Maybe a comment explaining it needs to be separated would help.

Reply:
Thanks for the hint. I'm removing `coverageAggregate` since we have a single module here.

Reply:
But please do not forget the other or we endup with broken coverage.

Reply:
Already added the comment to `codacy-plugins`. I will in other project I touch! Thank you 👍